website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -94,7 +94,6 @@ NRQL alert conditions support up to two terms. At least one `term` must have `pr
 
 The `term` block the following arguments:
 
-- `duration` - (Required) In minutes, must be in the range of `1` to `120`, inclusive.
```

Comment:
This attribute is deprecated, but still available right?

Reply:
Yep - it's already documented as deprecated lower in this file.

Reply:
https://github.com/newrelic/terraform-provider-newrelic/pull/760/files/82c1d051f10a07baa8905084c6eabf08a81d8fb0#diff-d2b2d7193559b0233ab2b133f2f591fcR105

Reply:
oh, derp. Thanks!